test/index.js

```diff
@@ -1,102 +1,125 @@
-import parse from '../lib/index.js';
-import {expect} from 'chai';
-import assert from 'assert';
-import { dirname } from 'path';
-import { fileURLToPath } from 'url';
+import parse from "../lib/index.js";
+import { expect } from "chai";
+import assert from "assert";
+import { dirname } from "path";
+import { fileURLToPath } from "url";
 
 const __dirname = dirname(fileURLToPath(import.meta.url));
-var POM_PATH = __dirname + "/fixture/pom.xml";
+const POM_PATH = __dirname + "/fixture/pom.xml";
 
-describe('require("pom-parser")', function () {
+//async functions are syntactic sugar for promises.
+//They never throw exceptions but may return rejected promise
+//Async function to handle the thrown exception from promise
+async function assertThrowsAsync(fn, regExp) {
+  let f = () => {};
+  try {
+    await fn();
+  } catch (e) {
+    f = () => {
+      throw e;
+    };
+  } finally {
+    assert.throws(f, regExp);
+  }
+}
 
-  describe('loading from files', function() {
-    var pomResponse = null;
-    var pom = null;
-    var xml = null;
+describe('require("pom-parser")', function () {
+  describe("loading from files", function () {
+    let pomResponse = null;
+    let pom = null;
+    let xml = null;
 
     // Setup the tests using mocha's promise.
     // https://lostechies.com/derickbailey/2012/08/17/asynchronous-unit-tests-with-mocha-promises-and-winjs/
-    before(function(done) {
-      parse({filePath: POM_PATH}, function(err, response) {
+    before(function (done) {
+      parse({ filePath: POM_PATH }, function (err, response) {
         expect(err).to.be.null;
         expect(response).to.be.an("object");
 
-	pomResponse = response;
-	pom = pomResponse.pomObject;
-	xml = pomResponse.pomXml;
-	done();
+        pomResponse = response;
```

Comment:
Exclude these pure formatting changes from this PR. We will do it as a separate PR.